nextflow_schema.json

```diff
@@ -93,6 +93,126 @@
+                    "fa_icon": "fas fa-border-all"
+                },
+                "annotation_bakta_gram": {
+                    "type": "string",
+                    "default": "?",
+                    "enum": ["+", "-", "?"],
+                    "description": "Specify the type of bacteria to be annotated to detect signaling peptides.",
+                    "help_text": "Specifies the type of bacteria expected in the input dataset for correct annotation of the signal peptide predictions. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--gram`",
+                    "fa_icon": "far fa-plus-square"
+                },
+                "annotation_bakta_complete": {
+                    "type": "boolean",
+                    "description": "Specify that all contigs are complete replicons.",
+                    "help_text": "This flag expects contigs that make up complete chromosomes and/or plasmids. By calling it, the user ensured that the contigs are complete replicons. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--complete`",
+                    "fa_icon": "far fa-circle"
+                },
+                "annotation_bakta_keepcontigheaders": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Keep original contig headers.",
+                    "help_text": "This flag specifies that the contig headers should not be rewritten. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--keep-contig-headers`",
+                    "fa_icon": "far fa-list-alt"
+                },
+                "annotation_bakta_compliant": {
+                    "type": "boolean",
+                    "description": "Clean the result annotations to standardise them to Genbank/ENA conventions.",
+                    "help_text": "Annotation in BAKTA is done by rapidly identifying known protein sequences with exact sequence matches and conducting comprehensive annotations using [PYRODIGAL](https://github.com/althonos/pyrodigal). BAKTA uses a compact read-only SQLite database consistig of protein sequence digests and pre-assigned annotations for millions of known protein sequences and clusters present in the database. More info can be found [here](https://github.com/oschwengers/bakta). The resulting annotations are cleaned up to standardise them to Genbank/ENA/DDJB conventions. CDS without any attributed hits and those without gene symbols or product descriptions different from hypothetical will be marked as 'hypothetical'.\nWhen activated the '--min-contig-length' must be set to 200.\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--compliant`\n",
```

Comment:
```suggestion
                    "help_text": "Annotation in BAKTA is done by rapidly identifying known protein sequences with exact sequence matches and conducting comprehensive annotations by internally using [PYRODIGAL](https://github.com/althonos/pyrodigal). BAKTA uses a compact read-only SQLite database consistig of protein sequence digests and pre-assigned annotations for millions of known protein sequences and clusters present in the database. More info can be found [here](https://github.com/oschwengers/bakta). The resulting annotations are cleaned up to standardise them to Genbank/ENA/DDJB conventions. CDS without any attributed hits and those without gene symbols or product descriptions different from hypothetical will be marked as 'hypothetical'.\nWhen activated the '--min-contig-length' must be set to 200.\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--compliant`\n",
```

Reply:
I don't understand what the first half of help_text is referring to, it looks to me at the moment it'sjust a genreal description of teh tool / Everything after 'more info can be found [here]' looks more useful to me :+1: